pom.xml

```diff
@@ -7,7 +7,8 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>rest-utils-parent</artifactId>
-        <version>[7.1.0-0, 7.1.1-0)</version>
+        <version>7.1.0-0</version>
+        <!-- <version>[7.1.0-0, 7.1.1-0)</version> -->
```

Comment:
Do we need to submit that temporarily? If yes, can you please add a comment explaining why? If not, is that a leftover from some testing?

Reply:
Yeah this is just so that it will consume the `rest-utils` local changes when running tests, I'll get rid of it before I convert this to a real PR.